dask_expr/_collection.py

```diff
@@ -591,6 +591,11 @@ def isna(self):
     def isnull(self):
         return new_collection(self.expr.isnull())
 
+    def mask(self, cond, other=np.nan):
```

Comment:
Note: IIUC, extension types would default to `pd.NA` so using `np.nan` isn't perfect.